src/data/use-cases/db-forget-password.ts

```diff
@@ -1,17 +1,24 @@
-import { type LoadGuardianByEmailRepository } from '@/data/protocols/guardian'
-import { type TokenGenerator } from '../protocols'
-import { type EmailService } from '@/domain/use-cases'
-import { type ForgetPassword } from '@/domain/use-cases/forget-password'
+import {
+  type LoadGuardianByEmailRepository,
+  type HashGenerator,
+  type UpdateVerificationTokenRepository,
+  type TokenGenerator
+} from '@/data/protocols'
+import { type EmailService, type ForgetPassword } from '@/domain/use-cases'
```

Comment:
domain imports could be done earlier to follow the pattern.

Reply:
What pattern? I'm using alphabetic order of paths.